odl/contrib/datasets/ct/mayo.py

```diff
@@ -0,0 +1,270 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Tomographic datasets from Mayo Clinic.
+
+In addition to the standard ODL requirements, this library also requires:
+
+    - tqdm
+    - dicom
+    - A copy of the Mayo dataset, see
+    https://www.aapm.org/GrandChallenge/LowDoseCT/#registration
+"""
+
+from __future__ import division
+import numpy as np
+import os
+import dicom
+import odl
+import tqdm
+
+from dicom.datadict import DicomDictionary, NameDict, CleanName
+from odl.contrib.datasets.ct.mayo_dicom_dict import new_dict_items
+
+# Update the dicom dictionary with the extra Mayo tags
+DicomDictionary.update(new_dict_items)
+new_names_dict = dict([(CleanName(tag), tag) for tag in
+                       new_dict_items])
```

Comment:
I don't think you need the outer list

Reply:
Good catch. Whole thing could be written

```
NameDict.update((CleanName(tag), tag) for tag in new_dict_items)
```